app/controllers/banneduser_experiment_controller.py

```diff
@@ -392,10 +392,15 @@ def _format_intervention_message(self, experiment_thing):
         condition = metadata_json["condition"]
         arm = metadata_json["arm"]
 
+        if condition not in self.experiment_settings["conditions"].keys():
+            raise ExperimentConfigurationError(
+                f"In the experiment '{self.experiment_name}', the '{condition}' condition fails to exist in the configuration"
+            )
+
         yml_cond = self.experiment_settings["conditions"][condition]
         if arm not in yml_cond["arms"].keys():
             raise ExperimentConfigurationError(
-                f"In the experiment '{self.experiment_name}', the '{self.get_condition()}' condition fails to include information about the '{arm}' arm, despite having randomizations assigned to it"
```

Comment:
good save; this would have been a different error because I'd renamed it to `_get_condition` and *that* method does checks redundant with what is happening here